functional-tests/README.md

```diff
@@ -51,8 +53,35 @@ Only on windows:
 
 Only on MacOS:
 
-    ./gradlew safariTest -DsafariTest.single=CustomJUnitSpecRunner
+    DEPRECATED ./gradlew safariTest -DsafariTest.single=CustomJUnitSpecRunner
+
+## Run tests in project root
+
+The tests can also be run from the projects root using the command 'make test'
```

Comment:
*project's

Probably should also mention what kind of test? Firefox/chrome, headless/normal?

Reply:
done